src/main/java/redis/clients/jedis/csc/CacheConfig.java

```diff
@@ -0,0 +1,69 @@
+package redis.clients.jedis.csc;
+
+public class CacheConfig {
+
+    public enum EvictionPolicyType {
+        DEFAULT, LRU
+    }
+
+    public enum CacheType {
+        DEFAULT, GUAVA, CAFFEINE
+    }
+
+    private int maxSize;
+    private EvictionPolicyType evictionPolicyType;
+    private ClientSideCacheable cacheable;
+    private CacheType cacheType;
+
+    public int getMaxSize() {
+        return maxSize;
+    }
+
+    public CacheType getCacheType() {
+        return cacheType;
+    }
+
+    public EvictionPolicyType getEvictionPolicyType() {
+        return evictionPolicyType;
+    }
+
+    public ClientSideCacheable getCacheable() {
```

Comment:
this is what i have in mind :point_down: 
https://redislabs.atlassian.net/wiki/spaces/CAE/pages/4295884832/Lightweight+client-side+caching+design?focusedCommentId=4480499735

Reply:
Wouldn't this information still be available to them if they call `COMMAND INFO`?

Reply:
it is more about managing it than getting info. 
handling the particular keys and/or commands and avoiding caching on user side.

Reply:
this enables user control with filtering keys and commands to cache or not,  as well as it will be useful in case we want to use the command tip "cacheable" from server in the future.